### PR TITLE
ES-96/SBOM generation

### PR DESCRIPTION
**Pull request by seanbrereton**
Creating SBOM using the cyclonedx plugin.
Adding a task to zip the sboms into 1 folder and publish them.
Tested locally: https://gradle.dev.r3.com/s/fhdql7ghlvoy4

### CODE REVIEW

**Review by yarivus — DISMISSED** [review]
LGTM

File: settings.gradle [hunk 1]
```diff
@@ -59,6 +59,7 @@ pluginManagement {
         id "com.jfrog.artifactory" version artifactoryPluginVersion
         id 'io.snyk.gradle.plugin.snykplugin' version snykVersion
         id 'com.github.ben-manes.versions' version dependencyCheckVersion
+        id "org.cyclonedx.bom" version "1.7.4"
```
Comment (ronanbrowne) [inline settings.gradle]: Can we externalize the hard-coded version to gradle.properties please

**Review by ronanbrowne — CHANGES_REQUESTED** [review]
See

**Review by ronanbrowne — APPROVED** [review]
LGTM

**Review by yarivus — APPROVED** [review]
LGTM

**Pull request merged**